lib/cucumber/formatter/console.rb

```diff
@@ -80,51 +78,23 @@ def print_element_messages(element_messages, status, kind)
         end
       end
 
-      def print_stats(features, options)
```

Comment:
The public methods of the Console module are part of the public API (see #893), so this PR introduces breaking changes (which is OK since we are aiming at 3.0.0 - but the PR is miss labeled).

Reply:
I would have been nice for this change to be documented in the changelog :beers: